data-serving/scripts/import-data/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "import-data",
+  "version": "1.0.0",
+  "description": "Creates a database and collection as necessary and imports data into it.",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "dev": "nodemon",
```

Comment:
Don't think we want nodemon for this type of script. This should probably just be `"start": "node dist/index.js"`.

Reply:
yeah I think we don't even want to run it with node, we want to run it with mongodb -- going to omit this line from this PR and add it in the next when i have a valid script